wrapper

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
-
-version="0.41.0"
+TMPDIR=/a/tmp export TMPDIR
+version="0.42"
 
 # --- Configuration variables ---
 
-# the filter executable:
-FILTER_BIN=/usr/bin/postconfirmc
-
-# Mailman's distributed mailman executable:
-MAILMAN_BIN=/usr/lib/mailman/mail/mailman
-
-# Mail handler executable
-SENDMAIL_BIN=/usr/sbin/sendmail
-
-# Where we store mailman commands identified as spam
-SPAM_STASH=/a/postconfirm/cache
+. wraper-param
```

Comment:
```suggestion
. wrapper-param
```

Reply:
If this was me I would have kept the defaults in the script, so that the values in `wrapper-param` were overrides.

Reply:
understood, but I think the structure in the rewrite should be different anyhow, so leaving this as is (except for fixing the typo)